.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
We don't have special environments for any other IDE... I don't like the idea of having a bunch of configs in the repo for IDEs. The only "special" treatment that we've given them is adding their custom files to 'gitignore', which I think is reasonable in order to aid development workflows. More than that is not okay by me.

Reply:
How about putting them in another repo then? Surely they should be shared somehow

Reply:
I agree we shouldn't endorse a particular editor or something like that. I do suggest to keep this `devcontainer.json` in the repository however, to support GitHub's [codespaces](https://docs.github.com/en/github/developing-online-with-codespaces/configuring-codespaces-for-your-project) once they launch. We can wait with adding the `devcontainer.json` to the repository until the GitHub feature has actually launched if people prefer. Here's the docs:

- https://docs.github.com/en/github/developing-online-with-codespaces/configuring-codespaces-for-your-project

Reply:
Fine with me. Once GitHub launches the feature it will be easy enough to transition.